.github/workflows/comment-on-pr.yml

```diff
@@ -0,0 +1,42 @@
+name: Comment on the pull request
+
+on:
+  workflow_run:
+    workflows: ["Integration tests"]
+    types:
+      - completed
+
+jobs:
+  upload:
+    runs-on: ubuntu-latest
+    if: >
+      github.event.workflow_run.event == 'pull_request' &&
```

Comment:
Shouldn't we update code coverage report only when PR is merged?

Reply:
This workflow is only run when the `integration tests` workflow completed (so also on each commit). Then it adds a comment to that PR providing the coverage result summary.

Reply:
OK